onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -198,6 +198,7 @@ def get_model(args: argparse.Namespace):
             source,
             torch_dtype=torch.float16 if args.use_fp16 else torch.float32,
             use_auth_token=args.auth,
+            trust_remote_code=True,
```

Comment:
May add an argument --trust_remote_code for this.

Reply:
Fixed by setting it to `args.auth` since `use_auth_token` and `trust_remote_code` aim to accomplish the same thing